tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -315,6 +315,11 @@ internal static partial class ConfigurationKeys
         /// <seealso cref="TracerSettings.ExpandRouteTemplatesEnabled"/>
         public const string ExpandRouteTemplatesEnabled = "DD_TRACE_EXPAND_ROUTE_TEMPLATES_ENABLED";
 
+        /// <summary>
+        /// Configuration key for enabling computation of metrics on the tracer side
+        /// </summary>
+        public const string TracerStatsEnabled = "DD_TRACE_TRACER_METRICS_ENABLED";
```

Comment:
Is the name `DD_TRACE_TRACER_METRICS_ENABLED` a standard across tracers? I don't see it used by any other tracer ([code search](https://cs.github.com/?scope=org%3ADataDog&scopeName=DataDog&q=repo%3ADataDog%2Fdd-trace-js+OR+repo%3ADataDog%2Fdd-trace-dotnet+OR+repo%3ADataDog%2Fdd-trace-java+OR+repo%3ADataDog%2Fdd-trace-rb+OR+repo%3ADataDog%2Fdd-trace-php+OR+repo%3ADataDog%2Fdd-trace-go+OR+repo%3ADataDog%2Fdd-trace-py+OR+repo%3ADataDog%2Fdd-opentracing-cpp+DD_TRACE_TRACER_METRICS_ENABLED)).

This name is confusingly close to `DD_TRACE_METRICS_ENABLED` (and the `..TRACE_TRACER..` bit is weird). Why not `DD_TRACE_STATS_ENABLED`? Trace "stats" have specific meaning for us., while "tracer metrics" is ambiguous.

If nothing else, can you at least move it up next to `DD_TRACE_METRICS_ENABLED` and `DD_RUNTIME_METRICS_ENABLED` so all the "metrics" settings are together?

Reply:
- runtime metrics = `DD_RUNTIME_METRICS_ENABLED`
- tracer metrics (aka tracer _health_ metrics) = `DD_TRACE_METRICS_ENABLED`
- trace metrics (aka trace stats) = ???

Reply:
`DD_TRACE_COMPUTE_STATS` is used by the Python tracer. While we haven't standardized this across the board, this name seems good to use

Reply:
Are the Go and Java team using `DD_TRACE_TRACER_METRICS_ENABLED`? If yes, Zach can you make sure that the name used by Python has been communicated to Go and Java? If we switch to this name, I assume we need to:
- Make sure we document this somewhere
- Create stories in Go and Java backlog to change the name.

Reply:
Here's what my digging has uncovered:

- Go: No specific environment variable, but only enabled when the `discovery` feature flag is set (for example via `DD_TRACE_FEATURES`)
- Java: `DD_TRACE_TRACER_METRICS_ENABLED`
- Python: `DD_TRACE_COMPUTE_STATS`

I posted internally so we can get a consensus.

Reply:
Here's what my digging has uncovered:

- Go: No specific environment variable, but only enabled when the `discovery` feature flag is set (for example via `DD_TRACE_FEATURES`)
- Java: `DD_TRACE_TRACER_METRICS_ENABLED`
- Python: `DD_TRACE_COMPUTE_STATS`

I posted internally so we can get a consensus.